docs/quickstart/docker.md

```diff
@@ -15,13 +15,13 @@ Coder with Docker has the following advantages:
 
 1.  [Install and launch Coder](../install)
 
-    You will specify `CODER_ACCESS_URL=http://localhost:7080` since we're using
+    Specify `CODER_ACCESS_URL=http://localhost:7080` since we're using
```

Comment:
add something like "after installing coder, we can quickly set up coder related variables by running the following command. This command will specify ...."